src/renderers/dom/client/utils/DOMChildrenOperations.js

```diff
@@ -247,6 +248,39 @@ var DOMChildrenOperations = {
     }
   },
 
+  startObservation: function() { 
+    var config = { attributes: false, childList: true, characterData: false, subtree: true };
+    var MutationObserver = window.MutationObserver || function() {};
+    MutationObserver.prototype.observe = MutationObserver.prototype.observe || function() {};      
+    MutationObserver.prototype.disconnect = MutationObserver.prototype.disconnect || function() {};      
```

Comment:
Here is an example where I added a simple devtool: https://github.com/facebook/react/pull/7040/files